index.bs

```diff
@@ -161,57 +161,81 @@ This is often used to attempt to trick users into visiting scam websites.
 If this feature was proposed today, it would probably not proceed.
 </p>
 
-<h3 id="consent">Ask users for meaningful consent</h3>
+<h3 id="user-intent">Design for user intent</h3>
 
```

Comment:
We somewhat failed to adhere to our own standards for principles with this one.  We're supposed to provide a simple, positive summary.  How about:

```suggestion

Ensure that it is possible to have the outcome of asking users
for permission to use exceptional capabilities
meet their expectations.

```

That wording is deliberate.  I'll suggest a note below that might make the reasoning clearer.

Reply:
Thanks for fixing that oversight of mine. I would love it if this statement could be a bit stronger, similar to my comment on the note you added at the end. Wdyt about something like this:

Use your API design to encourage websites to meet user expectations when accessing exceptional capabilities.

Reply:
That works for me.